dts/arm/atmel/same70.dtsi

```diff
@@ -419,7 +419,8 @@
 
 		can0: can@40030000 {
 			compatible = "atmel,sam-can";
-			reg = <0x40030000 0x100>;
+			reg = <0x40030000 0x100>, <0x40088110 0x04>;
```

Comment:
The size does not describe how many bytes is the register address, but how many bytes belong to the instance the address is pointing too.
In this case the size of the message ram.

Reply:
actually this is only 1 register (4byte)

Reply:
the size of the message ram is defined by 

`bosch,mram-cfg = <0x0 28 8 3 3 0 1 1>;`

Reply:
`0x40088110 ` is not the message ram; its a pointer for DMA that has to be filled with the start address of the message ram